components/TextControls/src/FilledTextAreas/MDCFilledTextArea.m

```diff
@@ -0,0 +1,80 @@
+// Copyright 2020-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "MDCFilledTextArea.h"
+
+#import <Foundation/Foundation.h>
+
+#import "MaterialTextControlsPrivate+FilledStyle.h"
+#import "MaterialTextControlsPrivate+Shared.h"
+
+@interface MDCFilledTextArea (Private) <MDCTextControl>
```

Comment:
Is this needed?

Reply:
Yes, without it the `self.containerStyle = ...` line won't compile